appveyor.yml

```diff
@@ -10,20 +10,21 @@ environment:
 # clone directory
 clone_folder: c:\projects\psscriptanalyzer
 
-# cache nuget packages
-cache:
-    - '%USERPROFILE%\.nuget\packages -> **\project.json'
```

Comment:
I am not sure how one would configure `NuGet` caching with the new csproj format...